src/internal/m365/collection/site/collection_test.go

```diff
@@ -282,66 +279,3 @@ func (suite *SharePointCollectionSuite) TestCollection_streamItems() {
 		})
 	}
 }
-
-// TestRestoreListCollection verifies Graph Restore API for the List Collection
-func (suite *SharePointCollectionSuite) TestListCollection_Restore() {
```

Comment:
were any lines changed with this movement?

Reply:
No @ryanfkeepers ,
This was moved to `/restore_test.go` under a new suite `SharePointRestoreSuite`.
The suite-setup and imports resulted in 70 additional lines.